File: backend/backend/src/main/java/com/moovie/backend/domain/model/MovieReviewDraft.java
package com.moovie.backend.domain.model;

import java.util.List;

public record MovieReviewDraft(
        String id,
        int movieId,
        String movieTitle,
        String posterPath,
        String reviewTitle,
        String reviewBody,
        double rating,
        boolean isFavorite,
        boolean isRewatch,
        List<String> tags
) {
}

File: src/main/kotlin/org/mobyle/domain/model/MovieReviewDraft.kt
package org.mobyle.domain.model

import kotlinx.serialization.Serializable

@Serializable
data class MovieReviewDraft(
    val id: String? = null,
    val movieId: Int,
    val movieTitle: String,
    val posterPath: String? = null,
    val reviewTitle: String,
    val reviewBody: String,
    val rating: Double,
    val isFavorite: Boolean = false,
    val isRewatch: Boolean = false,
    val tags: List<String> = emptyList()
)
